src/utils/axiosWithAuth.js

```diff
@@ -1,7 +1,11 @@
 import axios from 'axios';
 
 export const axiosWithAuth = () => {
-  const token = 'Bearer ' + localStorage.getItem('token');
```

Comment:
Thanks for fixing this, that was my fault, I copied the token so many times and never looked at what the token was actually called lol